NTPClient.h

```diff
@@ -44,6 +44,11 @@ class NTPClient {
      */
     void setPoolServerName(const char* poolServerName);
 
+     /**
+     * Set random local port
+     */
+    void setRandomPort(long minValue, long maxValue);
```

Comment:
Same here ... I think `int` or even better `unsigned int` (as it excludes negative values) would do the job just nicely here.

Reply:
I think you are right